specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/2016-08-10/service.json

```diff
@@ -9134,8 +9272,99 @@
         }
       }
     },
+    "VaultHealthDetails": {
+      "description": "Vault health details definition.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/VaultHealthProperties",
+          "description": "The vault health related data."
+        }
+      }
+    },
+    "VaultHealthProperties": {
+      "description": "class to define the health summary of the Vault.",
+      "type": "object",
+      "properties": {
+        "vaultErrors": {
+          "description": "The list of errors on the vault.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/HealthErrorSummary"
+          }
+        },
+        "protectedItemsHealth": {
+          "$ref": "#/definitions/ResourceHealthSummary",
+          "description": "The list of the health detail of the protected items in the vault."
+        },
+        "fabricsHealth": {
+          "$ref": "#/definitions/ResourceHealthSummary",
+          "description": "The list of the health detail of the fabrics in the vault."
+        }
+      }
+    },
+    "HealthErrorSummary": {
+      "description": "class to define the summary of the health error details.",
+      "type": "object",
+      "properties": {
+        "summaryCode": {
```

Comment:
Since this is the error code only which we have exposed at other classes as well, like health error, we don't feel this to be exposed as an enum hence going ahead with string.